templates/user/notification/notification_div.tmpl

```diff
@@ -59,24 +59,24 @@
 									{{end}}
 								</td>
 								<td class="eleven wide">
-									<a class="item" href="{{.Link}}">
+									<a class="item issue-title muted" href="{{.Link}}">
 										{{if $issue}}
-											#{{$issue.Index}} - {{$issue.Title}}
+											#{{$issue.Index}} - {{$issue.Title | RenderEmoji $.Context | RenderCodeBlock}}
 										{{else}}
 											{{$repo.FullName}}
 										{{end}}
 									</a>
 								</td>
 								<td>
-									<a class="item" href="{{$repo.Link}}">{{$repo.FullName}}</a>
+									<a class="item muted" href="{{$repo.Link}}">{{$repo.FullName}}</a>
 								</td>
 								<td class="collapsing">
 									{{if ne .Status 3}}
 										<form action="{{AppSubUrl}}/notifications/status" method="POST">
 											{{$.CsrfTokenHtml}}
 											<input type="hidden" name="notification_id" value="{{.ID}}">
 											<input type="hidden" name="status" value="pinned">
-											<button class="ui mini button" title='{{$.locale.Tr "notification.pin"}}'
+											<button class="ui mini button button-link" title='{{$.locale.Tr "notification.pin"}}'
```

Comment:
Actually, I think this is one of the places where the button can also look like a button.
Otherwise, it's really hard for a user to guess where they can click, and there is enough space that it doesn't increase the height/width of the containing element.

Reply:
Same below.

Reply:
Hmm we could add a hover highlight like GitHub has:

<img width="141" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/ca072147-7018-4fd2-b158-f0c2d5e04f6c">